apps/react/src/features/auth/components/UserProfile/UserProfile.tsx

```diff
@@ -0,0 +1,49 @@
+import { FC, memo } from 'react';
+import { Avatar, Box, Typography, Grid, CircularProgress } from '@mui/material';
+import { useAppSelector } from '@js-camp/react/store';
+import { selectUser, selectUserLoading } from '@js-camp/react/store/user/selectors';
+import { blue } from '@mui/material/colors';
+
+import styles from './UserProfile.module.css';
+
+const labelStyleSx = { sx: { color: blue[700] } };
```

Comment:
It's possible to do something like this: 
```suggestion
const labelStyleSx: SxProps = {
  color: blue[700]
}
```

Import `SxProps` from MUI :)